### ISSUE TITLE
`implements` clause is wrong

### ISSUE THREAD

**Issue by goodmind** [michaelgmcd/vscode-language-babel#55]
**Describe the issue**

`implements` clause highlighting is wrong

**Screenshot of Current Behavior**

![image](https://user-images.githubusercontent.com/3275424/62877596-cdcf9200-bd40-11e9-8802-0954a8f89511.png)

**Sample Code to Reproduce**

```js
interface A {
  a: string;
}
declare class Yo implements A {
  a: string;
}
```

**Comment by michaelgmcd** [comment]
This issue is due to the `declare` keyword so it's not a very common one. This was not supported in Flow until recently so it makes sense that this plugin wouldn't. I will take a look at this when I can.

**Comment by michaelgmcd** [comment]
This should be fixed in 0.0.25. Please let me know if it's not.
